doc/monitoring-stack-deployment/README.md

```diff
@@ -44,20 +133,43 @@ Then deploy prometheus server:
 oc apply -f prometheus.yaml
 ```
 
-6. Create Prometheus route
+8. Create Prometheus route
 
 ```bash
 oc expose service prometheus-operated --hostname prometheus.NAMESPACE_NAME.apps.CLUSTER_DOMAIN
 ```
 
-7. Deploy grafana datasource
+9. Deploy grafana datasource
 
 ```bash
-oc apply -f datasource.yaml
+oc apply -f datasource-v5.yaml
 ```
 
-8. Deploy grafana
+10. Deploy grafana
 
 ```bash
-oc apply -f grafana.yaml
+oc apply -f grafana-v5.yaml
 ```
+
+#### Migration from V4 to V5
+
```

Comment:
To correct:
1. Remove Grafana CR v4
2. Remove Grafana dataSource v4
3. Remove Grafana v4 Operator
4. Install Grafana v5 operator
5. Create Grafana datasource v5
6. Create Grafana CR v5
7. Restart 3scale operator
8. Optionally clean up dashboards v4 but restart 3scale operator again

Reply:
cc @carlkyrillos wdyt?

Reply:
@MStokluska These steps worked for me

Reply:
Updated, thanks